src/mono/mono/mini/local-propagation.c

```diff
@@ -841,7 +841,7 @@ static gboolean
 mono_is_simd_accessor (MonoInst *ins)
 {
 	switch (ins->opcode) {
-#ifdef MONO_ARCH_SIMD_INTRINSICS
+#if defined(MONO_ARCH_SIMD_INTRINSICS) && !defined(TARGET_ARM64) // FIXME: enable for Arm64
 	case OP_INSERT_I1:
```

Comment:
This needs to be fixed.

Reply:
It might be a good idea to reserve ARCH_SIMD_INTRINSICS for the existing code, and just unconditionally add the new code to simd-intrinsics-netcore.c, the define doesn't make too much sense any more.